src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import core.basesyntax.io.CsvReader;
+import core.basesyntax.io.CsvWriter;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.model.Report;
+
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private final CsvReader reader = new CsvReader();
+    private final CsvWriter writer = new CsvWriter();
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method name 'getStatistic' does not accurately reflect the action the method performs. Since it's writing a report to a file, a name like 'generateAndWriteReport' would be more informative.